### PR TITLE
Chat - last message group should have corresponding class

### PR DESCRIPTION
**Pull request by Zedwag**

### CODE REVIEW

File: packages/devextreme/js/__internal/ui/chat/messagelist.ts [hunk 1]
```diff
@@ -37,6 +37,11 @@ const CHAT_MESSAGELIST_EMPTY_MESSAGE_CLASS = 'dx-chat-messagelist-empty-message'
 const CHAT_MESSAGELIST_EMPTY_PROMPT_CLASS = 'dx-chat-messagelist-empty-prompt';
 const CHAT_MESSAGELIST_DAY_HEADER_CLASS = 'dx-chat-messagelist-day-header';
 
+const CHAT_MESSAGEGROUP_ALIGNMENT_START_CLASS = 'dx-chat-messagegroup-alignment-start';
+const CHAT_MESSAGEGROUP_ALIGNMENT_END_CLASS = 'dx-chat-messagegroup-alignment-end';
```
Comment (ksercs) [inline packages/devextreme/js/__internal/ui/chat/messagelist.ts]: looks like can be exported from message group

File: packages/devextreme/js/__internal/ui/chat/messagelist.ts [hunk 2]
```diff
@@ -361,10 +366,26 @@ class MessageList extends Widget<Properties> {
...
+      .removeClass(CHAT_LAST_MESSAGEGROUP_ALIGNMENT_START_CLASS);
+    this._$content
+      .find(`.${CHAT_LAST_MESSAGEGROUP_ALIGNMENT_END_CLASS}`)
+      .removeClass(CHAT_LAST_MESSAGEGROUP_ALIGNMENT_END_CLASS);
+
+    const $lastAlignmentStartGroup = this._$content.find(`.${CHAT_MESSAGEGROUP_ALIGNMENT_START_CLASS}`).last();
+    const $lastAlignmentEndGroup = this._$content.find(`.${CHAT_MESSAGEGROUP_ALIGNMENT_END_CLASS}`).last();
+
+    $lastAlignmentStartGroup.addClass(CHAT_LAST_MESSAGEGROUP_ALIGNMENT_START_CLASS);
+    $lastAlignmentEndGroup.addClass(CHAT_LAST_MESSAGEGROUP_ALIGNMENT_END_CLASS);
```
Comment (ksercs) [inline packages/devextreme/js/__internal/ui/chat/messagelist.ts:387]: well, it can be optimized ofc but I agree that's enough for now

**Review by ksercs — DISMISSED** [review]

File: packages/devextreme/js/__internal/ui/chat/messagelist.ts (hunk 1 above)
Comment (Zedwag) [reply packages/devextreme/js/__internal/ui/chat/messagelist.ts]: done

File: packages/devextreme/js/__internal/ui/chat/messagelist.ts (hunk 2 above)
Comment (Zedwag) [reply packages/devextreme/js/__internal/ui/chat/messagelist.ts:387]: good

Comment (EugeniyKiyashko) [reply packages/devextreme/js/__internal/ui/chat/messagelist.ts:387]: I don't like direct interaction with the markup. I don’t have any objections to leaving it as it is if it currently seems like an acceptable.

**Review by EugeniyKiyashko — DISMISSED** [review]

File: packages/devextreme/js/__internal/ui/chat/messagelist.ts [hunk 3]
```diff
@@ -24,7 +24,11 @@ import { getScrollTopMax } from '@ts/ui/scroll_view/utils/get_scroll_top_max';
 import { isElementVisible } from '../splitter/utils/layout';
 import type Chat from './chat';
 import type { MessageGroupAlignment } from './messagegroup';
-import MessageGroup from './messagegroup';
+import MessageGroup, {
+  CHAT_MESSAGEGROUP_ALIGNMENT_END_CLASS,
+  CHAT_MESSAGEGROUP_ALIGNMENT_START_CLASS,
+}
+  from './messagegroup';
```
Comment (EugeniyKiyashko) [inline packages/devextreme/js/__internal/ui/chat/messagelist.ts]: Strange formatting

Comment (Zedwag) [reply packages/devextreme/js/__internal/ui/chat/messagelist.ts]: oh right, ty

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**